schemas/pulse-job.yml

```diff
@@ -0,0 +1,277 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+title: "Job Definition"
+description: |
+  Definition of a single job that can be added to Treeherder
+  Project is determined by the routing key, so we don't need to specify it here.
+id: "jobDefinition"
+type: "object"
+properties:
+  jobGuid:
+    title: "jobGuid"
+    type: "string"
+    pattern: "^[A-Za-z0-9_/+-]+$"
+    minLength: 1
+    maxLength: 50
+
+  origin:
+    anyOf:
+      - type: "object"
+        properties:
+          kind:
+            type: "string"
+            enum: ['hg.mozilla.org']
+          project:
+            type: "string"
+            pattern: "^[A-Za-z0-9_-]+$"
+            minLength: 1
+            maxLength: 50
+          revision:
+            type: "string"
+            minLength: 1
+            maxLength: 50
+          pushLogID:
+            type: "integer"
+        required: [kind, project, revision]
+
+      - type: "object"
+        properties:
+          kind:
+            type: "string"
+            enum: ['github.com']
+          project:
```

Comment:
So I assume that "project" is the same thing as the repository.  Is there a place to define what user that project/repo is for?